### PR TITLE
make terms and conditions singular

### PR DESCRIPTION
**Pull request by FrontEndDev-Arhan**
more professional

### CODE REVIEW

**Comment by FrontEndDev-Arhan** [comment]
Studies have shown that writing Term and Condition rather than Terms and Conditions produce much better CTR's.

File: LICENSE [hunk 1]
```diff
@@ -2,7 +2,7 @@
                            Version 2.0, January 2004
                         http://www.apache.org/licenses/
 
-   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+   TERM AND CONDITION FOR USE, REPRODUCTION, AND DISTRIBUTION
```
Comment (Arhan-Kamra) [inline LICENSE:5]: Indeed, i guess you are right. But if you can create a little more space before and after 'AND' i would be happy to merge your request.

**Review by Arhan-Kamra — CHANGES_REQUESTED** [review]
If you make that change, i would be happy to merge.

File: LICENSE (hunk 1 above)
Comment (FrontEndDev-Arhan) [reply LICENSE:5]: sure, i will

Comment (Arhan-Kamra) [reply LICENSE:5]: excellent

**Pull request merged**